src/hardhat/constants/index.ts

```diff
@@ -0,0 +1,7 @@
+import { ethers } from "ethers";
+
+export const W3F_PATH = "./web3-functions/index.ts";
```

Comment:
doesn't seem to be used

Reply:
[2daa293](https://github.com/gelatodigital/web3-functions-sdk/pull/22/commits/2daa293e1e55eabba5d9ea2071022f9408e1701d)